src/widgetastic_patternfly4/card.py

```diff
@@ -42,7 +42,7 @@ class CardGroup( GenericLocatorWidget, View):
     
     def __init__(self, parent, locator=None, logger=None, **kwargs):
         View.__init__(self, parent, logger=logger, **kwargs)
-        self.locator = locator or '//section[@class="pf-c-page__main-section"]/div'
+        self.locator = locator or 'section[@class="pf-c-page__main-section"]/div'
```

Comment:
I believe it should be `self.locator = locator or './/section[@class="pf-c-page__main-section"]/div'`

Reply:
fixed, thanks